util/msgpack.go

```diff
@@ -0,0 +1,34 @@
+package util
```

Comment:
I think this file should be in `/internal/pkg`.

Reply:
Why? It doesn't import any convoy specific pkgs

Reply:
Oh, yes, then it should be `/pkg` directly, rather than `util`.